build/gulp/tasks/git/prepush.ts

```diff
@@ -0,0 +1,30 @@
+import { execSync } from 'child_process'
+
+// naive check is implemented for now - this checks if Semantic UI tags are present
+const hasWrongTags = (gitTagsList): boolean => {
+  // this one is guaranteed being relevant to Semantic UI only
+  const SemanticUiVersionTag = 'v0.1.2'
+
+  return gitTagsList.includes(SemanticUiVersionTag)
+}
+
+type Error = string
+
+export default (): Error => {
+  const gitTagsList = `${execSync('git tag -l')}`
+  if (hasWrongTags(gitTagsList)) {
+    const errorMessage = [
+      '!! YOUR LOCAL REPO CONTAINS WRONG GIT TAGS !!',
+      '-----------------------------------------------',
+      'To prevent wrong git tags from being pushed, please, consider to do the following steps:',
+      ' - delete all local tags: git tag -l | xargs git tag -d',
```

Comment:
Would you mind adding Windows instructions as well (there is no `xargs` on Windows)
```sh
FOR /f "tokens=*" %a in ('git tag -l') DO git tag -d %a
```
Source: https://gist.github.com/RandomArray/fdaa427878952d9768b0

Reply:
actually, these ones should work for Windows - have tested on my Windows 10 machine (thanks to WSL). But, for sure, if there will be any problem with them, will add Windows-specific ones